00_сompilation_process/00_02_link/00_02_00_link_error/00_02_00_second_source.cpp

```diff
@@ -0,0 +1,5 @@
+#include "000_header.h"
```

Comment:
What is `000_header.h`? I cannot find a header with this name. Also applicable for other includes.

Reply:
Done